sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -91,22 +91,23 @@ public class ChatClient {
         thread: CreateThreadRequest,
         withOptions options: Chat.CreateChatThreadOptions? = nil,
         completionHandler: @escaping HTTPResultHandler<CreateThreadResult>
-    ) {
+    ) throws {
         // Set the repeatabilityRequestID if it is not provided
-        let requestOptions = ((options?.repeatabilityRequestID) != nil) ? options : Chat.CreateChatThreadOptions(
-            repeatabilityRequestID: UUID().uuidString,
+        let requestOptions = ((options?.repeatabilityRequestId) != nil) ? options : Chat.CreateChatThreadOptions(
```

Comment:
The name change of `repeatabilityRequestID` to `repeatabilityRequestId` is a breaking change if it was previously released.